insights/parsers/dirsrv_logs.py

```diff
@@ -0,0 +1,97 @@
+'''
+389 Directory Server logs
+=========================
+
+'''
+
+from insights import parser, LogFileOutput
+
+
+@parser('dirsrv_access')
+class DirSrvAccessLog(LogFileOutput):
+    '''
+    The ``access`` log file from all directories in ``/var/log/dirsrv/``
+
+    This uses the standard LogFileOutput parser class for its implementation.
+
+    Note: This parser class gets the access file from all directories in
+    ``/var/log/dirsrv``.  Therefore, you will use this parser as a list in
+    the shared parsers dictionary.  Iterate through and check the file name
+    for the one you want, or scan all logs.
+
+    Sample input::
+
+            389-Directory/1.2.11.15 B2014.300.2010
+            ldap.example.com:636 (/etc/dirsrv/slapd-EXAMPLE-COM)
+
+        [27/Apr/2015:13:16:35 -0400] conn=1174478 fd=131 slot=131 connection from 10.20.10.106 to 10.20.62.16
+        [27/Apr/2015:13:16:35 -0400] conn=1174478 op=-1 fd=131 closed - B1
+        [27/Apr/2015:13:16:35 -0400] conn=324375 op=606903 SRCH base="cn=users,cn=accounts,dc=example,dc=com" scope=2 filter="(uid=root)" attrs=ALL
+        [27/Apr/2015:13:16:35 -0400] conn=324375 op=606903 RESULT err=0 tag=101 nentries=1 etime=0
+        [27/Apr/2015:13:16:35 -0400] conn=324375 op=606904 SRCH base="cn=groups,cn=accounts,dc=example,dc=com" scope=2 filter="(|(member=uid=root,cn=users,cn=accounts,dc=example,dc=com)(uniqueMember=uid=root,cn=users,cn=accounts,dc=example,dc=com)(memberUid=root))" attrs="cn"
+        [27/Apr/2015:13:16:35 -0400] conn=324375 op=606904 RESULT err=0 tag=101 nentries=8 etime=0
+        [27/Apr/2015:13:16:40 -0400] conn=1174483 fd=131 slot=131 connection from 10.20.130.21 to 10.20.62.16
+        [27/Apr/2015:13:16:40 -0400] conn=1174483 op=0 SRCH base="" scope=0 filter="(objectClass=*)" attrs="* altServer namingContexts aci"
+        [27/Apr/2015:13:16:40 -0400] conn=1174483 op=0 RESULT err=0 tag=101 nentries=1 etime=0
+
+
+    Examples:
+        >>> for access_log in shared[DirSrvAccessLog]:
+        ...     print "Path:", access_log.path
+        ...     connects = access_log.get('connection from')
+        ...     print "Connection lines:", len(connects)
+        ...
+        Path: /var/log/dirsrv/slapd-EXAMPLE-COM/access
+        Connection lines: 2
+    '''
+    pass
+    # def get_after(self, timestamp, lines=None):
```

Comment:
I'd really rather leave them there until the LogFileOutput get_after work in #118 and #119 gets merged, then I'll update these and remove the comments.  I commented the code out just to test it against the branches without that addition.